widgets/google-map/js/js-map.js

```diff
@@ -404,6 +404,30 @@ jQuery(function ($) {
 				apiUrl += '&key=' + apiKey;
 			}
 
+			// This allows us to "catch" Google Maps JavaScript API errors and do a bit of custom handling. In this case,
+			// we display a user-specified fallback image if there is one.
+			if ( window.console && window.console.error ) {
+				var errLog = window.console.error;
+
+				sowb.onLoadMapsApiError = function ( error ) {
+					var matchError = error.match( /^Google Maps API (error|warning): ([^\s]*)\s([^\s]*)(?:\s(.*))?/ );
```

Comment:
You can see a list of the errors and warnings (about halfway down the list) [here](https://developers.google.com/maps/documentation/javascript/error-messages#deverrorcodes)